src/Components/News/NewsForm.js

```diff
@@ -78,17 +78,18 @@ const NewsForm = ({ mode = "create", novelity }) => {
     };
     //If the mode is "create", the api is called via the POST verb, if not, the PUT verb is called with ID of novelity
     if (mode === "create") {
-      await axios.post("http://ongapi.alkemy.org/api/news", dataObject, {
-        "Content-Type": "application/json",
-      });
+      try {
+        await postNews(dataObject)
+      } catch (error) {
+        error && setError(true)
```

Comment:
Si entra al catch quiere decir que error existe, no hay que validarlo.

```suggestion
        setError(true)
```